### PR TITLE
adjust changelog layout and update contributing details.

### LINKED ISSUE #1029: CHANGELOG Process
Research and recommend a CHANGELOG process for Ark.

Examples:
  * https://github.com/restic/restic/tree/master/changelog
  * https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md

### PR DESCRIPTION
**Pull request by wwitzel3**
closes #1029 
Signed-off-by: Wayne Witzel III <wayne@riotousliving.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 # Contributing
+
+## Updating CHANGELOG.md
+
+Authors are expected include an entry to the CHANGLOG.md under the `Development release` heading
```
Comment (skriss) [inline CONTRIBUTING.md]: s/CHANGLOG.md/CHANGELOG.md, could include a link as well

File: changelogs/CHANGELOG-0.9.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  - https://github.com/heptio/ark/releases/tag/v0.9.11
+
+### Bug Fixes
+  * Fix bug preventing PV snapshots from being restored (#1040, @ncdc)
+
+
+## v0.9.10
+#### 2018-11-01
+### Download
+  - https://github.com/heptio/ark/releases/tag/v0.9.10) - 2018-11-01
```
Comment (skriss) [inline changelogs/CHANGELOG-0.9.md]: remove `) - 2018-11-01`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,326 +1,22 @@
...
-
-#### [v0.3.1](https://github.com/heptio/ark/tree/v0.3.1) - 2017-08-03
-  * Fix Makefile VERSION
-
-#### [v0.3.0](https://github.com/heptio/ark/tree/v0.3.0) - 2017-08-03
-  * Initial Release
-
-[1]: https://github.com/heptio/ark/blob/master/changelogs/CHANGELOG-0.10.md
+## Development release:
+
```
Comment (skriss) [inline CHANGELOG.md]: it would probably be helpful to have a template here so contributors know what format their release note should be in

**Comment by skriss** [comment]
close/reopen to kick signoff check

**Comment by skriss** [comment]
@nrb @carlisia @ncdc pls review - let's get this in soon so we can start including changelog entries with each PR

**Comment by skriss** [comment]
one other note - we'll want to make sure we have release notes under Development release for any PRs that have gone in since v0.10.  @wwitzel3 can you do that in a separate PR once this has gone in?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,4 +1,11 @@
 # Contributing
+
+## Updating CHANGELOG.md
+
+Authors are expected include an entry to the CHANGELOG.md under the `Development release` heading
+with their pull requests. During the process of creating a new release entries from the root changelog will be
```
[the remark this answers is not in the document]
Comment (wwitzel3) [reply CONTRIBUTING.md]: This will be manual. We can maybe look at adding tooling later if we find it to onerous.

**Comment by wwitzel3** [comment]
@skriss Yeah, I can do a PR after this lands for our current development features.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,4 +1,11 @@
 # Contributing
+
+## Updating CHANGELOG.md
+
+Authors are expected include an entry to the CHANGELOG.md under the `Development release` heading
+with their pull requests. During the process of creating a new release, entries from CHANGELOG.md will be.
```
Comment (skriss) [inline CONTRIBUTING.md]: remove end of line `.`

**Comment by skriss** [comment]
LGTM, ready to merge

**Pull request merged**